presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -570,6 +566,24 @@ public static OptionalLong fromMetastoreNullsCount(long nullsCount)
         return OptionalLong.of(nullsCount);
     }
 
+    /**
+     * Impala 'COMPUTE STATS' write 1 as the numTrue and -1 as the numFalse
+     * @see <a href="https://github.com/apache/impala/blob/0cbe37afc8f77fd8e0aef7e710e2d1b214c1c5b1/fe/src/main/java/org/apache/impala/catalog/ColumnStats.java#L269">Impala ColumnStats.java#L269</>
```

Comment:
Please put this comment to under the `if` in 577 line. It is very related to `-1` literal there, not the method as a whole.

Reply:
OK